build-server/src/main/resources/dockerfiles/java-maven/Dockerfile

```diff
@@ -12,7 +11,7 @@ RUN add-apt-repository ppa:natecarlson/maven3
 RUN apt-get update && apt-get -y upgrade
 
 # Automatically accept oracle license
-RUN echo oracle-java8-installer shared/accepted-oracle-license-v1-1 select true | sudo /usr/bin/debconf-set-selections
+RUN echo oracle-java8-installer shared/accepted-oracle-license-v1-1 select true | /usr/bin/debconf-set-selections
```

Comment:
Needed to change this to get the Dockerfile to build on my Docker version (1.4.0)